src/nft/components/collection/Card.tsx

```diff
@@ -256,6 +256,7 @@ const Image = ({ uniformHeight, setUniformHeight }: ImageProps) => {
         width="full"
         style={{
           aspectRatio: uniformHeight === UniformHeights.notUniform ? '1' : 'auto',
+          minHeight: uniformHeight,
```

Comment:
I think this change had an unintended effect as the aspect ratio shift seems to happen more frequently. More details sent in DM